src/System.Data.Common/tests/System/Data/Common/DataColumnMappingCollectionTest.cs

```diff
@@ -548,5 +548,27 @@ public void Remove_Object_PassingNullThrowsException()
 
             AssertExtensions.Throws<ArgumentNullException>("value", () => dataColumnMappingCollection.Remove((object)null));
         }
+
+        [Fact]
+        public void This_Int_SetAndGetOK()
```

Comment:
@stephentoub 

Thanks for the feedback.  I'll update it the next time that I add more tests.

Michael